### DISCUSSION TITLE
Get macOS to recognize fonts?

### DISCUSSION THREAD

**Discussion by fenneltheloon** [TomSchimansky/CustomTkinter#357] [General]
I would like to use custom fonts, packaged with the project and referenced by path. However, when I include them in my theme.json, macOS raises a big stink in the console and uses the default system fonts. 

```
CoreText note: Client requested name "./fonts/Roboto/Roboto-Regular.ttf", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
CoreText note: Set a breakpoint on CTFontLogSystemFontNameRequest to debug.
CoreText note: Client requested name "./fonts/Roboto/Roboto-Regular.ttf", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
```

Above is an example error thrown. Is there any current way to load custom fonts, or even get macOS to use any font other than system even if it means installing it, as that didn't seem to be working either.

**Comment by TomSchimansky** [comment]
You can not just insert a path to use a font. You have to load the custom font file into the system to be able to use it.
Have a look here: https://stackoverflow.com/questions/11993290/truly-custom-font-in-tkinter
You can try to use `pyglet.font.add_file` but I had problems with it on macOS in combination with using tkinter.
